src/pages/organisationUnitLevels/ListRows.tsx

```diff
@@ -0,0 +1,251 @@
+                                offlineLevels:
+                                    selected === ''
+                                        ? undefined
+                                        : parseInt(selected),
+                            })
+                        }}
+                    >
+                        <SingleSelectOption
+                            key={`offlineLevel-none`}
+                            value={''}
+                            label={i18n.t('Default')}
+                        />
+                        {Array.from({ length: 15 }, (_, i) => i + 1).map(
+                            (level) => (
+                                <SingleSelectOption
+                                    key={`offlineLevel-${level}`}
+                                    value={level.toString()}
+                                    label={level.toString()}
+                                />
+                            )
+                        )}
+                    </SingleSelect>
+                )
+            }
+            if (path === 'displayName' && editingOrgUnitLevel !== undefined) {
+                return (
+                    <Input
+                        value={editingOrgUnitLevel.name || ''}
+                        width={'100px'}
+                        onChange={(event) =>
+                            setEditingOrgUnitLevel({
+                                ...editingOrgUnitLevel,
+                                ['name']: event.value || '',
+                            })
+                        }
+                    />
+                )
+            }
+            return (
+                <ModelValue path={path} schema={schema} sectionModel={model} />
+            )
+        },
+        [schema]
+    )
+
+    return (
+        <SectionList
+            headerColumns={headerColumns.map((column) => ({
+                ...column,
+                disableSorting: true,
+            }))}
+        >
+            <SectionListMessage />
+            {orgUnitLevels?.map((model, index) => (
+                <DataTableRow
+                    className={css.listRow}
+                    dataTest={`section-list-row-${model.id}`}
+                    key={model.id}
+                >
+                    <DataTableCell width="48px" />
+                    {headerColumns.map((selectedColumn) => (
+                        <DataTableCell key={selectedColumn.path}>
+                            {renderColumnValue({
+                                path: selectedColumn.path,
+                                model,
+                                editingOrgUnitLevel:
+                                    index === editingOrgUnitLevel?.index
+                                        ? editingOrgUnitLevel
+                                        : undefined,
+                            })}
+                        </DataTableCell>
+                    ))}
+                    <DataTableCell>
```

Comment:
This will fix most of the layout-shift when editing. 
```suggestion
                    <DataTableCell width='150px'>
```